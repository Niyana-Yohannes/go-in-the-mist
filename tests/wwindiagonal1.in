place A1
place A19
place B1
place B18
place C1
place C17
place D1
place D16
place A2
place E15
