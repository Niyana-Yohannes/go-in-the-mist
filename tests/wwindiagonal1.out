White wins!
A1A19B1B18C1C17D1D16A2E15
Thank you for playing!
